apps/docs/Dockerfile

```
@@ -1,50 +1,45 @@
 RUN adduser --system --uid 1001 nextjs
 USER nextjs
 
-# Automatically leverage output traces to reduce image size
-# https://nextjs.org/docs/advanced-features/output-file-tracing
 COPY --from=builder --chown=nextjs:nodejs /app/apps/docs/.next/standalone ./
 COPY --from=builder --chown=nextjs:nodejs /app/apps/docs/.next/static ./apps/docs/.next/static
 COPY --from=builder --chown=nextjs:nodejs /app/apps/docs/public ./apps/docs/public
 
-CMD node apps/web/server.js --port 3000
\ No newline at end of file
+CMD node apps/docs/server.js
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)